apps/wallet/src/ui/app/shared/faucet/useFaucetMutation.ts

```diff
@@ -26,15 +26,16 @@ export function useFaucetMutation(options?: UseFaucetMutationOptions) {
 				throw new Error('Failed, faucet host not found.');
 			}
 
-			const { error, transferredGasObjects } = await requestSuiFromFaucetV0({
+			// based on requestSuiFromFaucetV1 response type, we no longer get the amount transferred
+			const { error } = await requestSuiFromFaucetV1({
 				recipient: addressToTopUp,
 				host: options.host,
 			});
 
 			if (error) {
 				throw new Error(error);
 			}
-			return transferredGasObjects.reduce((total, { amount }) => total + amount, 0);
+			return null;
```

Comment:
It seems that the new API returns a task id with this request. That then is used to check if the request was successful. So it might take a while before the faucet actually sends the tokens and also it might fail but we will still show success.

Should we add here the check for the task status and then return?

Reply:
Or we could change the messaging and say something like `Request submitted` (but this will break a bit the button and allow multiple requests)

Reply:
Let me update this PR

Reply:
Made copy change, I don't think `getFaucetRequestStatus` works appropriately; it is a get request in Rust, but the SDK has it as POST.

Reply:
Should we update the api when it's ready then? Why to add patches now?